x/authz/client/testutil/query.go

```diff
@@ -28,7 +28,7 @@ func (s *IntegrationTestSuite) TestQueryAuthorizations() {
 			fmt.Sprintf("--%s=100stake", cli.FlagSpendLimit),
 			fmt.Sprintf("--%s=true", flags.FlagSkipConfirmation),
 			fmt.Sprintf("--%s=%s", flags.FlagFrom, val.Address),
-			fmt.Sprintf("--%s=%s", flags.FlagBroadcastMode, flags.BroadcastBlock),
+			fmt.Sprintf("--%s=%s", flags.FlagBroadcastMode, flags.BroadcastSync),
```

Comment:
I think some e2e tests are failing because of these changes, we might need to do `s.network.WaitForNextBlock()` each time